appveyor.yml

```diff
@@ -0,0 +1,15 @@
+version: '{build}'
+build_script:
+- cmd: >-
+    set RUBY_VERSION=200
+
+    set PATH=C:\Ruby%RUBY_VERSION%\bin;%PATH%
+
+    gem install bundler
+
+    bundle install
+
+    build.netstd.cmd
+test: off
+on_finish:
+    - ps: $root = Resolve-Path .\artifacts; [IO.Directory]::GetFiles($root.Path, '*.*', 'AllDirectories') | % { Push-AppveyorArtifact $_ -FileName $_.Substring($root.Path.Length + 1) -DeploymentName to-publish }

```

Comment:
Yes, exactly as @adamralph says. Clean.log and Build.log in artifacts/logs.
I actually had a mysterious failure on the CI server and had insufficient hints about what went wrong until I added this step.